src/server/req_modify.c

```diff
@@ -815,17 +819,19 @@ req_modifyReservation(struct batch_request *preq)
 			reply_badattr(PBSE_ATTRRO, 1, psatl, preq);
 			return;
 		}
-
+		
 		switch (index) {
 			case RESV_ATR_start:
 				if ((presv->ri_wattr[RESV_ATR_state].at_val.at_long != RESV_RUNNING) || !num_jobs) {
 					temp = strtol(psatl->al_value, &end, 10);
-					if ((temp > time(NULL)) &&
-						(temp != presv->ri_wattr[RESV_ATR_start].at_val.at_long)) {
+					if (temp > time(NULL)) {
 						if (!is_standing || (temp < next_occr_start)) {
 							send_to_scheduler = RESV_START_TIME_MODIFIED;
 							presv->ri_alter_stime = presv->ri_wattr[RESV_ATR_start].at_val.at_long;
 							presv->ri_alter_flags |= RESV_START_TIME_MODIFIED;
+							pdef->at_decode(&presv->ri_wattr[RESV_ATR_start], 
+							psatl->al_name, psatl->al_resc, psatl->al_value);
```

Comment:
I dont think you should be doing this, if the scheduler rejects the alter, the start time will still change, right?

Reply:
Have you addressed this comment?